db/migrate/20170504213048_remove_postal_code_from_targets.rb

```diff
@@ -0,0 +1,23 @@
+class RemovePostalCodeFromTargets < ActiveRecord::Migration
+  class Call < ActiveRecord::Base; end
+  class Plugins::CallTool < ActiveRecord::Base; end
+
+  def up
+    batch_size = 300
+    Call.find_in_batches(batch_size: batch_size) do |group|
+      group.each do |call|
+        target = call.read_attribute(:target)
+        target.delete('postal_code')
+        call.update!(target: target)
+      end
+    end
+
+    Plugins::CallTool.find_in_batches(batch_size: batch_size) do |group|
+      group.each do |call_tool|
+        targets = call_tool.read_attribute(:targets)
+        targets.each { |t| t.delete('postal_code') }
+        call_tool.update!(targets: targets)
+      end
+    end
+  end
+end
```

Comment:
I've nothing against the script, but I don't think we should wrap in a migration. Migrations are intended for schema changes only. It's going to be executed every time someone (re) runs Champaign's migrations), which is brittle.